### PR TITLE
docs(relnotes): FX116 release notes offset-path and ray()

### PR DESCRIPTION
**Pull request by dipikabh**
### 1. offset-path
- **Previous syntax:** `offset-path: none | ray( [ <angle> && <size> && contain? ] ) | <path()> | <url> | [ <basic-shape> || <geometry-box> ] `
- **New syntax:** `offset-path: none | <offset-path> || <coord-box>`
where:
    `<offset-path> = <ray()> | <url> | <basic-shape>`
    `ray( <angle> && <size>? && contain? && [at <position>]? )`
- MDN page: https://developer.mozilla.org/en-US/docs/Web/CSS/offset-path
- Page updates and new `<box-edge>` page that documents `<coord-box>`: https://github.com/mdn/content/pull/28348
- Prefs required to be enabled in stable Fx116 for `<basic-shape>` and `<coord-box>` parameters:
  - `layout.css.motion-path-basic-shapes.enabled`  
  - `layout.css.motion-path-coord-box.enabled`
- Bugs:
  - https://bugzilla.mozilla.org/show_bug.cgi?id=1598156
  - https://bugzilla.mozilla.org/show_bug.cgi?id=1837305

### 2. ray()
- **Previous syntax:** `ray( <angle> && <ray-size>? && contain? )`
- **New syntax:** `ray( <angle> && <size>? && contain? && [at <position>]? )`
- MDN page: https://developer.mozilla.org/en-US/docs/Web/CSS/ray
- Page updates: https://github.com/mdn/content/pull/28348
- Pref required to be enabled in stable Fx116 for `ray()`: `layout.css.motion-path-ray.enabled`
- Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1820070 (ray)

The previous `ray()` release note was captured in the 112 release note file. Because this function is still available only behind preference, I've removed the entry from the release notes file and added it to the 'Experimental features' file.

### Related issues and pull requests

Doc issues:
- https://github.com/mdn/content/issues/27747
- https://github.com/mdn/content/issues/27751

Content PR: https://github.com/mdn/content/pull/28348

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/116/index.md [hunk 1]
```diff
@@ -15,15 +15,9 @@ This article provides information about the changes in Firefox 116 that affect d
...
-#### Removals
-
-No notable changes.
-
 ### CSS
 
-#### Removals
-
-No notable changes
+- The syntax of the {{cssxref("offset-path")}} property, which is used to define the path for an element to follow, has been updated. The `<offset-path>` parameter in the syntax consolidates the parameters `<ray()>`, `<url>`, and `<basic-shape>`. Additionally, the [`<coord-box>`](/en-US/docs/Web/CSS/box-edge) parameter, which lets you specify the shape of the path based on the element's box model, has replaced the previous `<geometry-box>` parameter in the syntax. The `none` keyword is still the default value for the property. The `<basic-shape>` and `<coord-box>` parameters require the `layout.css.motion-path-basic-shapes.enabled` and `layout.css.motion-path-coord-box.enabled` preferences, respectively, to be enabled. ([Firefox bug 1598156](https://bugzil.la/1598156)) and ([Firefox bug 1837305](https://bugzil.la/1837305)).
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/116/index.md:20]: Is this correct? What do you think?

```suggestion
- The syntax has been updated for the {{cssxref("offset-path")}} property, which is used to define the path for an element to follow in CSS motion path animations. The new syntax allows you to set a value of `none` or one of `<offset-path>` or `<coord-box>`. The new `<offset-path>` value can be a `<ray()>`, `<url>`, or a `<basic-shape>`. The [`<coord-box>`](/en-US/docs/Web/CSS/box-edge) value has replaced the older `<geometry-box>` value and lets you specify the shape of the path based on the element's box model. The `<basic-shape>` and `<coord-box>` values require the `layout.css.motion-path-basic-shapes.enabled` and `layout.css.motion-path-coord-box.enabled` preferences to be enabled, respectively ([Firefox bug 1598156](https://bugzil.la/1598156)) and ([Firefox bug 1837305](https://bugzil.la/1837305)).
```

**Review by bsmth — APPROVED** [review]
One comment outstanding based on my understanding of https://drafts.fxtf.org/motion/#offset-path-property, but everything looks good, so leaving my +1 👍🏻

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 2]
```diff
@@ -349,6 +349,50 @@ The {{cssxref("backdrop-filter")}} property applies filter effects to the area b
...
+    <tr>
+      <th>Release channel</th>
+      <th>Version added</th>
+      <th>Enabled by default?</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>Nightly</th>
+      <td>72</td>
```
[the remark this answers is not in the document]
Comment (dipikabh) [reply files/en-us/mozilla/firefox/experimental_features/index.md:371]: Good call. Thank you for digging up that bug!

File: files/en-us/mozilla/firefox/releases/116/index.md (hunk 1 above)
Comment (dipikabh) [reply files/en-us/mozilla/firefox/releases/116/index.md:20]: Thank you for the edits. The rewrite is much clearer.

**Pull request merged**